### DISCUSSION TITLE
A node installed observable/inputs table seems to break the height?

### DISCUSSION THREAD

**Discussion by bbkane** [observablehq/framework#1852] [Q&A] [answered]
Hello again! After https://github.com/observablehq/framework/discussions/1840 , I started trying to replace `npm:...` imports with node installed imports. However, when I tried to do this with `npm:@observablehq/inputs`, the height of the resulting table overflowed the header beneath it.

This is probably easy to explain with an example, so I made https://github.com/bbkane/framework-input-tables/tree/master

After running `npm run dev`, I see the following differences:

---

```js
import * as Inputs from "npm:@observablehq/inputs";
```

![image](https://github.com/user-attachments/assets/b24d4166-323c-45f5-8ded-6fdae045c111)

This works as expected

---

However, I see the following after running `npm install @observablehq/inputs` and trying to use it:

```js
import * as Inputs from "@observablehq/inputs";
```

![image](https://github.com/user-attachments/assets/9da7e775-d65d-4e7f-bb0c-14937b6f8050)

---

I'm not sure why it matters which way I import the library, and I'm curious if one of you could explain what I'm doing wrong with the local import :) . Is this something to do with the [import rules](https://observablehq.com/framework/imports) ?

**Comment by Fil** [comment, marked as answer]
This module also needs a stylesheet, which is transparently added when you use npm.
https://github.com/observablehq/framework/blob/8efdcf45c00f7cb471deab01f65fe93e1b539d44/src/libraries.ts#L63

You can also load it from npm independently of the node module:
```
<link rel=stylesheet href="npm:@observablehq/inputs/dist/index.css">
```

However I'm not sure there is a _simple_ way to load it from the node module directly
https://observablehq.com/framework/imports#node-imports

**Reply by bbkane** [reply]
Interesting, thanks for linking that file! I'll have to keep those implicit imports in mind when writing my code. Would you be up for a small docs PR to the imports page that mentions how this works?
